README.md

```diff
@@ -69,6 +69,6 @@ outline init
 
 Detailed information regarding how to contribute to Outline development, can be found in [CONTRIBUTING.md](./CONTRIBUTING.md). **Expect this information to be refined soon.**
 
-Please see [./docs/README.md](./docs/README.md) for information about developing for Outline.
+Please see [Component Development Guide](.packages/outline-docs/src/guides/development/component-development) for information about developing for Outline.
```

Comment:
`.packages/outline-docs/src/guides/development/component-development` isn't a valid path. See the [full file](https://github.com/phase2/outline/blob/affc18d1a7ca6a304813f97f0563afcb6967a6ef/README.md) in GitHub to see the link location.